### PR TITLE
Cache does not store current site in the cache key (#76)

### PR DESCRIPTION
**Pull request by MakeilaLundy**

### CODE REVIEW

File: src/controllers/DefaultController.php [hunk 1]
```diff
@@ -77,6 +76,7 @@ public function actionIndex(string $pattern): Response
             $plugin = Plugin::getInstance();
             $config = $plugin->getEndpoint($pattern);
             $request = Craft::$app->getRequest();
+            $site = Craft::$app->getSites()->getCurrentSite()->id;
```
Comment (angrybrad) [inline src/controllers/DefaultController.php]: Let's change this variable name to `$siteId`

File: src/controllers/DefaultController.php [hunk 2]
```diff
@@ -87,7 +87,7 @@ public function actionIndex(string $pattern): Response
             $cache = ArrayHelper::remove($config, 'cache', false);
 
             if ($cache) {
-                $cacheKey = 'elementapi:'.$request->getPathInfo().':'.$request->getQueryStringWithoutPath();
+                $cacheKey = 'elementapi:'.$request->getPathInfo().':'.$request->getQueryStringWithoutPath().':'.$site;
```
Comment (angrybrad) [inline src/controllers/DefaultController.php]: Let's move the `$siteId` part of the key to after `elementapi:` and before the path info so it's going from a wider scope to a narrower one.

**Comment by angrybrad** [comment]
@MakeilaLundy @Radabaugh after addressing the comments, can you add a line to CHANGELOG.md for this as well?

**Comment by MakeilaLundy** [comment]
yeah i will thanks!

**Pull request merged**